internal/entity/album.go

```diff
@@ -639,6 +650,15 @@ func (m *Album) UpdateFolder(albumPath, albumFilter string) error {
 	} else if err = UnscopedDb().Exec("UPDATE albums SET album_path = NULL WHERE album_type = ? AND album_path = ? AND id <> ?", AlbumFolder, albumPath, m.ID).Error; err != nil {
 		return err
 	}
+	
+	// update folder
+	if folder := FindFolder("/", albumPath); folder != nil {
```

Comment:
The `Updates()` function used above should automatically refresh the `update_at` column, so updating it again should not be necessary.